packages/Tooltip/styles.ts

```diff
@@ -2,9 +2,43 @@ import styled, { css, system, th } from '@xstyled/styled-components'
 import { Tooltip as ReakitTooltip } from 'reakit/Tooltip'
 import { filterSystemProps } from '@welcome-ui/system'
 
+import { PlacementOptions, TooltipOptions } from './index'
+
 export const Tooltip = styled(ReakitTooltip).withConfig({ shouldForwardProp: filterSystemProps })(
-  () => css`
-    ${th('tooltips')};
-    ${system};
-  `
+  () => css``
 )
+const transformDirection = {
+  top: 'translate3d(0, -4px, 0)',
+  right: 'translate3d(4px, 0, 0)',
+  bottom: 'translate3d(0, 4px, 0)',
+  left: 'translate3d(-4px, 0, 0)',
+  auto: 'translate3d(0, 0, 0)',
+}
+type TransformDirectionKey = keyof typeof transformDirection
+
+export const FadeIn = styled.div<{
+  placement?: PlacementOptions
+  fixed?: TooltipOptions['fixed']
+}>`
+  ${th('tooltips')};
+  ${system};
+  background-color: rgba(33, 33, 33, 0.9);
```

Comment:
get from theme if possible

Reply:
background-color is given on the Tooltip figma -> https://www.figma.com/file/TC5aahnPVDU1hhBcff037e/Feedback?node-id=2%3A890